docs/src/layouts/AppFooter.tsx

```diff
@@ -25,7 +25,7 @@ export default function AppFooter() {
           gridAutoColumns: '1fr',
           alignItems: 'center',
           justifyContent: 'space-between',
-          gap: (theme) => theme.spacing(4, 2),
+          gap: (theme) => theme.spacing(4),
```

Comment:
would have been simpler:

```suggestion
          gap: 4,
```